rllib/evaluation/sampler.py

```diff
@@ -49,6 +49,15 @@
 StateBatch = List[List[Any]]
 
 
+class NewEpisodeDefaultDict(defaultdict):
```

Comment:
Had to create this helper class to be able to pass it the key that's actually being queried for an entry.